### DISCUSSION TITLE
notificaciones en telegram

### DISCUSSION THREAD

**Discussion by Hoskargl** [ellite/Wallos#255] [Ideas]
Buenas tardes.
Me gustaría saber si es posible implementar que los avisos o notificaciones se pudiesen hacer en telegram. He instalado la aplicación y creo que sólo es posible a través de email.
Gracias

**Comment by ellite** [comment]
Hola. Nuevos métodos de notificación están en mi lista para ser implementados.

Saludos.

**Reply by Hoskargl** [reply]
Muchas gracias!

**Comment by guhanmathi** [comment]
Would you please add a feature to use Telegram to send notification.
I would like to share a project where telegram notifications are available. Please add this feature if possible. 
(https://github.com/Casvt/MIND)

**Comment by ellite** [comment]
The new notification agents are now live:
- Telegram
- Discord
- Gotify
- Pushover
- Webhooks

**Reply by guhanmathi** [reply]
Reply, thanks a lot.. Your work is much appreciated.
